### DISCUSSION TITLE
Is QtWebsockets supported?

### DISCUSSION THREAD

**Discussion by ohmree** [danvratil/qcoro#62] [Q&A] [answered]
See title.

**Comment by danvratil** [comment]
Currently not, but I should be able to implement it in the next release.

**Reply by danvratil** [reply]
https://github.com/danvratil/qcoro/issues/64

**Comment by danvratil** [comment, marked as answer]
Hi, the code has been merged in #75. I took a bit more work than I anticipated in the end as I had to implemented generators to QCoro. In any case, the code is now available in the main branch.

I still have to write docs for the `QCoroWebSockets` wrapper, but it should be fairly straightforward to understand (and the documentation for generators is available on qcoro.dvratil.cz)
